templates/crd-servicedefaults.yaml

```diff
@@ -89,6 +89,13 @@ spec:
               protocol:
                 description: Protocol sets the protocol of the service. This is used by Connect proxies for things like observability features and to unlock usage of the service-splitter and service-router config entries for a service.
                 type: string
+              transparentProxy:
+                description: TransparentProxy controls configuration specific to proxies in transparent mode.
+                properties:
+                  outboundListenerPort:
+                    description: The port of the listener where outbound application traffic is being redirected to.
+                    type: integer
+                type: object
```

Comment:
wondering if our docs should say that we don't support these?

Reply:
It doesnt say it in the docs but we do have a webhook that rejects any CRD that has this set with an error message that says: Please you annotation <annotation-name>`
I went back and forth on this a little bit and decided to only use the webhook because I figured it would be confusing for the docs to say that it wasn't supported. Do you think it makes sense to also have it in the docs?

Reply:
Hmm yeah, I can see that. Let's keep it as is then.

Reply:
I added this as a part of the commit that adds Mode and UpstreamConfigs to ServiceDefaults. Will update these CRDs as a part of their equivalent helm commit if that is ok?